ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/common/worldview/accumulator/DiffBasedWorldStateUpdateAccumulator.java

```diff
@@ -357,12 +471,6 @@ public void commit() {
                 pendingStorageUpdates.clear();
               }
 
-              // This is especially to avoid unnecessary computation for withdrawals and
```

Comment:
is this related to the feature? is so why?

Reply:
There was a side effect because we manipulate and commit the Accumulator a lot more because of the background thread. Thanks to this we saw a bug that impacted the performance of this PR and also Besu in general. This line of code prevents the purge of entries of the map which are useless because there is a clean that was no longer done. This results in larger map to check and an impact on performance